components/notifier.rst

```diff
@@ -1,33 +0,0 @@
-.. index::
```

Comment:
Please add a redirect for this URL as it was already public.

Reply:
Good catch, redirection added